src/createManager.js

```diff
@@ -72,7 +72,7 @@ const protoManager = {
 
   openMenu(openOptions) {
     if (this.isOpen) return;
-    openOptions = openOptions || {};
+    openOptions = openOptions || { focusMenu: true };
```

Comment:
I think this isn't a safe way to assign defaults — because what if `openOptions` is defined but does not specify `focusMenu`? (Right now it's the only option, but in the future it may not be.)